src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -72,12 +72,12 @@ public CommandResult execute(ModuleList moduleList, Configuration configuration)
             Module targetModule = moduleList.getGeneralTasks();
             if (!Objects.isNull(targetModuleName)) {
                 targetModule = moduleList.getModule(targetModuleName);
-                if (Objects.isNull(targetModule)) {
-                    throw new NoSuchModuleException();
-                }
+            }
+            if (Objects.isNull(targetModule)) {
+                throw new NoSuchModuleException();
             }
             TaskList taskList = targetModule.getTaskList();
-            res = String.format(ADD_TASK_MESSAGE, targetModule, taskList.addTask(newTask), taskList.size());
```

Comment:
Does this mean that the size of tasks in list will not be shown?

Reply:
Yes, I think we agreed to remove the message as it does not add value to user